ilastik/applets/tracking/conservation/opConservationTracking.py

```diff
@@ -328,7 +328,11 @@ def _resolveMergers(self, hypothesesGraph, model):
             withFullGraph = True
             logger.info("Computing full graph on merger resolver (Only enabled on animal tracking workflow)")
 
-        mergerResolver = IlastikMergerResolver(originalGraph, pluginPaths=self.pluginPaths, withFullGraph=withFullGraph)
+        # supply random_state for reproducible merger resolving
+        # random_state will be used in gmm-based merger resolving
+        mergerResolver = IlastikMergerResolver(
+            originalGraph, pluginPaths=self.pluginPaths, withFullGraph=withFullGraph, random_state=42
```

Comment:
Move `random_state` to a keyword parameter in `track()`? It probably doesn't matter, but the `parameters` dict already contains a lot of configurable values with defaults.

Reply:
you mean like in future we (users) might want to modify the seed? E.g. on a per project basis?

Reply:
I'm not sure to what extent that would be useful. Just thought about lifting hardcoded constants up to the config. But this is probably unnecessary in this case.

Reply:
hmmm.... on the _other_ hand...